server/datastore/mysql/scripts_test.go

```diff
@@ -1592,37 +1592,47 @@ func testDeletePendingHostScriptExecutionsForPolicy(t *testing.T, ds *Datastore)
 	require.NoError(t, err)
 	require.Equal(t, 1, len(pending))
 
-	// TODO(mna): adjust test once script execution via unified queue is implemented
```

Comment:
Unrelated to the Android work, a forgotten piece of test that needed uncommenting/adjusting after unified queue work.